lib/repo/git_hooks/client/pre-commit.d/04-file_size_too_large.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env sh
+
+CHANGED_FILES=$(git diff --name-status --no-renames --diff-filter=d HEAD) # diff does not include deleted files
+MAX_FILE_SIZE=$(git show HEAD:.max_file_size)
+
+# shellcheck disable=SC2034
+echo "$CHANGED_FILES" | while read -r _status path; do
+  sha=$(git ls-files --stage "$(git rev-parse --show-toplevel)/$path" | cut -d' ' -f 2)
+  size=$(git cat-file -s "$sha")
+  if [ "$size" -gt "$MAX_FILE_SIZE" ]; then
+    echo "[MARKUS] Error: The size of the uploaded file $path exceeds the maximum of $MAX_FILE_SIZE bytes."
```

Comment:
These aren't changed\s you introduced, but if you don't mind making these small changes:

1. Change `[MARKUS]` to `[MarkUs]` to be consistent
2. Change "uploaded" to "modified" (since "upload" has more of a "push" connotation)